<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(99, 99, 99);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid red;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../recursos.qrc">:/meme/Imagen/fians.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>520</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../recursos.qrc">:/meme/Imagen/logo_equipo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../recursos.qrc">:/meme/Imagen/uat-logo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Calcularsueldo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Calcular sueldo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_numempleados">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>90</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Num. de empleados:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_hrs">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>240</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Horas Trabajadas:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pagoxhora">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Extra Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>380</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pago por hora:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
